public/acquisition.html

```diff
@@ -151,6 +151,8 @@ <h3 id="sessionIdValue">Acquisition Helper</h3>
     <script type="text/javascript" src="https://www.gstatic.com/firebasejs/8.10.1/firebase-app.js" ;></script>
     <script src="https://www.gstatic.com/firebasejs/8.10.1/firebase-database.js"></script>
     <script>
+        //Why is this inside a script tag, instead of being in its own file or something
```

Comment:
this is inside a script tag on its own because it is from the Firebase documentation. I'm sure this block could be moved to its own file, but I'll run some tests to make sure that is the case